.travis.yml

```
@@ -111,6 +111,56 @@ matrix:
         before_script: skip
         after_script: skip
 
+    - name: Robot Framework Tests
+      #if: env(TRAVIS_EVENT_TYPE) = cron
```

```suggestion
      if: env(TRAVIS_EVENT_TYPE) = cron
```

Commit before merging.